client/scss/components/_panel.scss

```diff
@@ -1,6 +1,9 @@
 $header-icon-size: theme('spacing.4');
 $header-button-size: theme('spacing.6');
 
+@use '../tools' as *;
+@use '../settings' as *;
```

Comment:
For cases like this, I think it would be more idiomatic to have the imports first, then local module variables.